src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.scss

```diff
@@ -60,3 +57,41 @@ $content-width: $gene_image_width;
   justify-content: center;
   margin-top: 40px;
 }
+
+.xrefWithChevron {
+  display: flex;
+}
+
+.xrefGroupChevron {
+  text-align: right;
+  padding-right: 2px;
+  cursor: pointer;
+  font-weight: $normal;
+  min-width: 50px;
+}
+
+.chevron {
+  margin-left: 6px;
+  margin-bottom: -1px;
+  height: 12px;
+  width: 12px;
+}
+
+.chevronUp {
+  transform: rotate(0deg);
+}
+
+.chevronDown {
+  transform: rotate(-180deg);
+}
+
+.xrefsWrapper {
+  padding-bottom: 10px;
+  display: flex;
+  flex-direction: column;
+}
+
+.xrefGroupWrapper {
+  display: flex;
+  flex-direction: column;
```

Comment:
I don't think these two CSS rules are doing anything. Getting stacked on top of one another is the default behaviour of elements with display: block, which is the display property of this wrapper's children.

By the way, why is there an xrefGroupWrapper inside another xrefGroupWrapper?

![image](https://user-images.githubusercontent.com/6834224/116410615-dcb7ea00-a82c-11eb-919e-740e1fc39833.png)